### PR TITLE
Features/food orders

### PR DESCRIPTION
**Pull request by vjumpkung**
Follwing this features is finished and reviewable
- create database schema 
- create POST request /orders

Disclaimer : this endpoint is not finished because wait table collection and menu collection
- List Orders by session [/sessions/:id/orders]
```ts
response.table = randomInt(1, 10); //wait Table
response.total_price = randomInt(1, 1000); // wait Menu
```
this following two lines is only mock data for test response only

example response
```json
{
    "table": 9,
    "total_price": 91,
    "discount_price": 0,
    "net_price": 91,
    "orders": [
        {
            "_id": "64bea1949f1fe25d5d00368f",
            "created_at": "2023-07-24T16:06:13.142Z",
            "status": "IN_QUEUE",
            "session": "64b95d8869ac259060ac2c6e",
            "menu": "64b963027d0c7729a48d8f67",
            "addons": [
                "64b963047d0c7729a48d8f6b",
                "64b963047d0c7729a48d8f6b"
            ],
            "additional_info": "xxx",
            "__v": 0
        },
        {
            "_id": "64bea1949f1fe25d5d003690",
            "created_at": "2023-07-24T16:06:13.142Z",
            "status": "IN_QUEUE",
            "session": "64b95d8869ac259060ac2c6e",
            "menu": "64b963047d0c7729a48d8f6f",
            "addons": [
                "64b963047d0c7729a48d8f6b",
                "64b963047d0c7729a48d8f6d"
            ],
            "additional_info": "bbb",
            "__v": 0
        },
        {
            "_id": "64bea3059c7b3a1c566692ba",
            "created_at": "2023-07-24T16:12:11.533Z",
            "status": "IN_QUEUE",
            "session": "64b95d8869ac259060ac2c6e",
            "menu": "64b963027d0c7729a48d8f67",
            "addons": [
                "64b963047d0c7729a48d8f6b",
                "64b963047d0c7729a48d8f6b"
            ],
            "additional_info": "yyyyyyyyyxxx",
            "__v": 0
        },
        {
            "_id": "64bea3059c7b3a1c566692bb",
            "created_at": "2023-07-24T16:12:11.533Z",
            "status": "IN_QUEUE",
            "session": "64b95d8869ac259060ac2c6e",
            "menu": "64b963047d0c7729a48d8f6f",
            "addons": [
                "64b963047d0c7729a48d8f6b",
                "64b963047d0c7729a48d8f6d"
            ],
            "additional_info": "bbbxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
            "__v": 0
        }
    ]
}
```

### CODE REVIEW

**Comment by vjumpkung** [comment]
Problems list : 
- in /docs it's cannot add id as params and make id is editable
- 
image
should I use string or ObjectId becasue annoying '{' <- It's think it is object

File: package.json [hunk 1]
```diff
@@ -13,7 +13,6 @@
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\"",
-    "lint:fix": "eslint \"{src,apps,libs,test}/**/*.ts\"",
```
Comment (ishiasaka) [inline package.json:16]: Why are u remove this script?

File: package.json [hunk 2]
```diff
@@ -25,15 +24,14 @@
     "@nestjs/common": "^10.0.0",
     "@nestjs/config": "^3.0.0",
     "@nestjs/core": "^10.0.0",
+    "@nestjs/mongoose": "^10.0.0",
     "@nestjs/platform-express": "^10.0.0",
     "@nestjs/platform-fastify": "^10.0.5",
     "@nestjs/swagger": "^7.1.1",
-    "@typegoose/typegoose": "^11.4.0",
```
Comment (ishiasaka) [inline package.json:31]: Why delete it?

File: package.json [hunk 3]
```diff
@@ -25,15 +24,14 @@
...
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
     "firebase-admin": "^11.9.0",
-    "mongoose": "7.4.0",
-    "nest-typegoose": "^1.1.1",
```
Comment (ishiasaka) [inline package.json:36]: Why delete it?

File: package.json [hunk 4]
```diff
@@ -25,15 +24,14 @@
...
+    "mongoose": "^7.3.4",
```
Comment (ishiasaka) [inline package.json:34]: Why downgrade?

File: src/schema/categories.schema.ts [hunk 5]
```diff
@@ -1,9 +1,9 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
 import { ApiProperty } from '@nestjs/swagger';
-import { Prop, modelOptions } from '@typegoose/typegoose';
 import { Types } from 'mongoose';
 
-@modelOptions({ schemaOptions: { collection: 'categories' } })
```
Comment (ishiasaka) [inline src/schema/categories.schema.ts:5]: Use typegoose pls.

**Review by ishiasaka — CHANGES_REQUESTED** [review]

**Comment by vjumpkung** [comment]
This branch will abandon because I forgot to pull new develop and idk what's happen so I will create new branch instead

**Pull request closed without merging**